### PR TITLE
Notify build failed on slack

### PR DESCRIPTION
**Pull request by mharsat**
Click below to open examples:
rich-content: https://rich-content-notify-build-failed-on-slack.surge.sh/
rich-content-storybook: https://rich-content-storybook-notify-build-failed-on-slack.surge.sh/

### CODE REVIEW

File: .github/workflows/actions.yml [hunk 1]
```diff
@@ -88,4 +88,20 @@ jobs:
...
+  notify-on-slack:
+    runs-on: ubuntu-18.04
+    needs: [visual-tests, content-state-tests, unit-tests, lint]
+    if: always()
+    steps:
+      - name: Workflow Conclusion Action
+        uses: technote-space/workflow-conclusion-action@v1.1.1
+      - name: Notify on Slack
+        uses: voxmedia/github-action-slack-notify-build@v1.1.1
+        if: env.WORKFLOW_CONCLUSION == 'fail' && github.ref == 'refs/heads/notify-build-failed-on-slack'
```
Comment (oded-soffrin) [inline .github/workflows/actions.yml:101]: this probably need to be changed before merging to master, right ?

File: .github/workflows/actions.yml [hunk 2]
```diff
@@ -88,4 +88,20 @@ jobs:
...
+        with:
+          channel_id: GFT1VFDJ6 #rich-content-core
```
Comment (oded-soffrin) [inline .github/workflows/actions.yml:103]: I suggest to start with a dedicated channel, to make sure it doesn't spam us. #rich-content-ci ?

**Comment by oded-soffrin** [comment]
Big like, added some comments on related lines

File: .github/workflows/actions.yml (hunk 1 above)
Comment (mharsat) [reply .github/workflows/actions.yml:101]: whoops I was sure I changed it :)

File: .github/workflows/actions.yml [hunk 3]
```diff
@@ -25,4 +25,3 @@ jobs:
       - run: npm run deploy
     env:
       SURGE_LOGIN: ${{ secrets.SURGE_LOGIN }}
-      SURGE_TOKEN: ${{ secrets.SURGE_TOKEN }}
```
Comment (oded-soffrin) [inline .github/workflows/actions.yml:28]: is this related?

Comment (mharsat) [reply .github/workflows/actions.yml:28]: just a duplicate line

Comment (mharsat) [reply .github/workflows/actions.yml]: whoops
